### ISSUE TITLE
Python 3.6 is missing from ubuntu 22.04 for arch x64

### ISSUE THREAD

**Issue by rouilj** [actions/setup-python#561]
**Description:**

Similar to issue #543 which addresses Python version 2.7, version 3.6 is also missing.
```
Run actions/setup-python@v4
  
Version 3.6 was not found in the local cache
Error: Version 3.6 with arch x64 not found
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```
and 3.6 is missing from the manifest.

**Action version:**

4

**Platform:**
- [x] Ubuntu
- [ ] macOS
- [ ] Windows

**Runner type:**
- [x] Hosted
- [ ] Self-hosted

**Tools version:**

3.6

**Repro steps:**  

See: 

   https://github.com/roundup-tracker/roundup/actions/runs/3617160920/jobs/6127223518

**Expected behavior:**

I expect version 3.6 to run on ubuntu 22.04

**Actual behavior:**

environment setup fails

**Comment by rouilj** [comment]
As mentioned on issue 543, 3.6 is used for a number of long term releases.
 https://github.com/actions/setup-python/issues/543#issuecomment-1339592610

**Comment by davidpellcb** [comment]
We just ran into this issue with 3.9.10:

```
Run actions/setup-python@v2
Version 3.9.10 was not found in the local cache
Error: Version 3.9.10 with arch x64 not found
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```

**Comment by hugovk** [comment]
3.6 is a duplicate of https://github.com/actions/setup-python/issues/544, close reason (https://github.com/actions/setup-python/issues/544#issuecomment-1332535877):

> Hello everyone. Python 3.6 [reached end-of-life on Dec 12, 2021](https://peps.python.org/pep-0494/#lifespan). Therefore, we won’t be supporting it with new versions of our hosted runners. If you need Python 3.6, it’s still available on `ubuntu-20.04`. There are a couple reasons we stop supporting end-of-life versions. They no longer have maintenance support, so there’s no path forward for fixing bugs or security issues. Also, they tend to run into compat issues with dependencies on newer operating systems, which is a source of confusing bugs.
> 
> For now I'm going to close the issue.

3.9.10 is a duplicate of https://github.com/actions/setup-python/issues/553:

> Python 3.9.10 is built for Ubuntu18, Ubuntu20. You can try to switch to ubuntu-20.04 tag.

**Comment by MaksimZhukov** [comment]
Hello @davidpellcb  !
The `ubuntu-latest` tag is being migrated to Ubuntu 22.04 (see details in the related [announcement](https://github.com/actions/runner-images/issues/6399)). We have not built Python `3.9.10` for Ubuntu 22.04. Python 3.9 is available for installation on Ubuntu 22.04 starting with Python `3.9.12`. Could you please switch to the Python `3.9.12` (or higher) or specify Python version as `3.9` to install the latest available patch version?
```yml
    - name: Set up Python
      uses: actions/setup-python@v4
      with:
        python-version: '3.9'
```

**Comment by MaksimZhukov** [comment]
I am closing this issue as a duplicate of [this one](https://github.com/actions/setup-python/issues/544).
